docs/en/index.rst

```diff
@@ -126,18 +126,17 @@ Queue the messages using the included `Queue\QueueManager` class::
     use App\Job\ExampleJob;
     use Cake\Queue\QueueManager;
 
-    $callable = [ExampleJob::class, 'execute'];
     $data = ['id' => 7, 'is_premium' => true];
     $options = ['config' => 'default'];
 
-    QueueManager::push($callable, $data, $options);
+    QueueManager::push(ExampleJob::class, $data, $options);
 
 Arguments:
 
-- ``$callable``: A callable that will be invoked. This callable **must** be valid
-  within the context of your application. Job classes are prefered.
-- ``$data`` (optional): A json-serializable array of data that will be passed to
-  the job via message. It should be key-value pairs.
+- ``$callable``: The class that will have it's execute method invoked when the
```

Comment:
```suggestion
- ``$callable``: The class that will have its execute method invoked when the
```

Reply:
I wouldn't call it `callable` if it's a class name.

Reply:
I did have it named `$class` but we also accept the format `[class, method]` to maintain backwards compatibility for now so  I thought keeping the same name would be fine until we remove the backward compatibility. If you have a recommendation for a better name I would be happy to update it.

Reply:
I would call it `className` since that's the expected value and the deprecated value will  be removed.

Reply:
Updated to `className`.